<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Analyse and Integrate</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Pictures/Screenshot from 2016-12-07 15-03-21.png</normaloff>../../../Pictures/Screenshot from 2016-12-07 15-03-21.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,2">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2">
         <item>
          <layout class="QHBoxLayout" name="listContainer"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Reanalyse Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Output PDF for Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Ouptut CSV for selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>CollateAll PDFs in the Output Folder</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,2">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QGroupBox" name="ImageGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Image</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,2,1" columnstretch="3,1">
            <item row="1" column="1" rowspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2"/>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="imageLayout"/>
            </item>
            <item row="2" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4"/>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget" native="true"/>
            </item>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="sumRowsLayout"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Analysis</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1" columnstretch="0,1">
            <item row="0" column="1">
             <widget class="QTableWidget" name="tableWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QVBoxLayout" name="verticalLayout"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHere">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionQuit"/>
    <addaction name="actionHere"/>
    <addaction name="actionChange_Project_Settings"/>
   </widget>
   <addaction name="menuHere"/>
  </widget>
  <action name="actionHere">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Open Existing Project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionChange_Project_Settings">
   <property name="text">
    <string>Change Project Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
